.github/workflows/build-desktop-apps.yml

```diff
@@ -75,6 +75,15 @@ jobs:
             latest*.yml
           retention-days: 3
 
+      - name: Upload suite-desktop production artifacts
```

Comment:
Change name to describe that it contains arm file

Reply:
118aab4f06df9355ef31462c836d2ea6370f66aa